pkg/plugininventory/sqlite_inventory.go

```diff
@@ -674,7 +674,12 @@ func (b *SQLiteInventory) InsertPluginGroup(pg *PluginGroup, override bool) erro
 		}
 	}
 
-	allowHiddenPlugins, _ := strconv.ParseBool(os.Getenv(constants.ConfigVariableIncludeDeactivatedPluginsForTesting))
+	includeDeactivatedPluginsForTesting, _ := strconv.ParseBool(os.Getenv(constants.ConfigVariableIncludeDeactivatedPluginsForTesting))
+	activatePlugins, _ := strconv.ParseBool(os.Getenv(constants.ActivatePluginsOnPluginGroupPublish))
```

Comment:
Yeah. I think it is good to keep this check to avoid activating plugins while publishing a deactivated plugin group.

I don't see a use case to activate plugins when publishing a plugin-group in a deactivated state. I will make this change.